pkg/sys/unshare_linux.go

```diff
@@ -81,7 +81,7 @@ func UnshareAfterEnterUserns(uidMap, gidMap string, unshareFlags uintptr, f func
 		if dupErr != nil {
```

Comment:
If you still need to push again (for the sign-off); could you update this one as well? (`dupPidfd, err := `). Sorry I thought I commented on this one, but looks like I forgot to submit before I had to jump into a meeting 🙈 


(changes LGTM otherwise ❤️)